README.md

```diff
@@ -24,6 +24,8 @@ A generic makefile for starting projects
 
     words here
 
+    Added words because i'm bored
```

Comment:
Please add more words. 

:)

Reply:
Even more words added to the README.md file